docs/_static/css/custom.css

```diff
@@ -174,3 +174,62 @@ img[src="_images/snake_dark.svg"] {
   max-height: 0;
   overflow: hidden;
 }
+
+.appearance-selector-separator {
+  border: none;
+  border-top: 1px solid var(--color-sidebar-search-border);
+  margin: 0.5rem 0;
+}
+
+.width-selector-label {
+  font-weight: normal !important;
+  margin-top: 0 !important;
+}
+
+.width-option {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  cursor: pointer;
+  font-size: 0.85em;
+  color: var(--color-foreground-primary);
+  font-family: "Outfit", sans-serif;
+  transition: color 0.2s ease;
+  justify-content: start !important;
+  width: 100% !important;
+  padding: var(--sidebar-item-spacing-vertical) var(--sidebar-item-spacing-horizontal);
+  padding-top: 0;
+  padding-bottom: 0;
+  padding-right: 0;
+}
+
+.width-option:hover {
+  color: var(--color-brand-content);
+}
+
+.width-option input[type="radio"] {
+  cursor: pointer;
+  margin: 0;
+  width: 1em;
+  height: 1em;
+}
+
+.width-option span {
+  user-select: none;
+  position: static;
+  display: inline;
+  flex-shrink: 0;
+}
+
+.sidebar-tree .width-option {
+  display: flex !important;
+  position: relative !important;
```

Comment:
I'm not too sure about the importance of these importance markers :p

Reply:
It breaks if I don't put them; also they are scoped to a custom class of ours so unlikely it touches anything else